FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/statemachine/DistanceAutonomous.java

```diff
@@ -53,9 +40,8 @@ public MotorOutput driveTrainSpeed() {
 	}
 	
 	public double getDistanceTraveled() {
-		return distanceTraveled;
+		return encoder.getDistance();
```

Comment:
This is the wrong way to do it. What if `encoder.getDistance()` is returning a non-zero number when we begin?